test/models/actions-inputs-model.test.js

```diff
@@ -1,61 +1,79 @@
 const actionsInputsModel = require('../../server/models/action-inputs-model')
-const allParcels = [{ ref: 'AAA', description: 'Test parcel', totalPerimeter: 10, totalArea: 1 }]
+const allParcels = [{ ref: 'SD74445738', description: 'Test parcel', totalPerimeter: 10, totalArea: 1 }]
 
 describe('actions inputs model', () => {
+  const actions = {
+    fg6: {
+      id: 'FG1',
+      input: {
+        unit: 'metres',
+        description: 'fence length'
+      }
+
+    },
+    sw6: {
+      id: 'SW6',
+      input: {
+        unit: 'hectares',
+        description: 'winter crops area'
+      }
+
+    }
+  }
   test('if error provided, should populate errorMessage', () => {
     const error = 'sample error message'
-    const { model } = actionsInputsModel('', {}, error)
+    const { model } = actionsInputsModel('SD74445738', actions.fg6, error)
```

Comment:
I'd have been tempted to leave the parcel ref empty, as we're not asserting anything on that, we're focused on the error. The same goes for all the subsequent statements where the parcel ref is unimportant.

Reply:
Good spot. Removed.